{
    "active_lock_reason": null,
    "assignee": null,
    "assignees": [],
    "author_association": "MEMBER",
    "body": "**Description of the problem**\r\n\r\nSee the results of the \"GMT Latest Tests\" action (https://github.com/GenericMappingTools/pygmt/actions?query=workflow%3A%22GMT+Latest+Tests%22).\r\n\r\nThere are 10 failures on Windows. The error messages are:\r\n```\r\n_____________ ERROR at setup of test_grdview_grid_dataarray[png] ______________\r\n\r\nregion = (-116, -109, -47, -44)\r\n\r\n    @pytest.fixture(scope=\"module\", name=\"xrgrid\")\r\n    def fixture_xrgrid(region):\r\n        \"\"\"\r\n        Load the xarray.DataArray grid from the sample earth_relief file.\r\n        \"\"\"\r\n>       return grdcut(grid=\"@earth_relief_01d_g\", region=region)\r\n\r\nC:\\Miniconda3\\envs\\test\\lib\\site-packages\\pygmt\\tests\\test_grdview.py:34: \r\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\r\nC:\\Miniconda3\\envs\\test\\lib\\site-packages\\pygmt\\helpers\\decorators.py:267: in new_module\r\n    return module_func(*args, **kwargs)\r\nC:\\Miniconda3\\envs\\test\\lib\\site-packages\\pygmt\\helpers\\decorators.py:411: in new_module\r\n    return module_func(*args, **kwargs)\r\nC:\\Miniconda3\\envs\\test\\lib\\site-packages\\pygmt\\gridops.py:116: in grdcut\r\n    return result\r\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\r\n\r\nself = <pygmt.helpers.tempfile.GMTTempFile object at 0x0000019EBCD6D190>\r\nargs = (None, None, None)\r\n\r\n    def __exit__(self, *args):\r\n        if os.path.exists(self.name):\r\n>           os.remove(self.name)\r\nE           PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\\\Users\\\\RUNNER~1\\\\AppData\\\\Local\\\\Temp\\\\pygmt-xqghaxyy.nc'\r\n\r\nC:\\Miniconda3\\envs\\test\\lib\\site-packages\\pygmt\\helpers\\tempfile.py:70: PermissionError\r\n```\r\n\r\nIt's unclear if it's a bug in GMT dev version or a PyGMT bug.",
    "closed_at": "2021-02-10T20:47:15Z",
    "closed_by": {
        "avatar_url": "https://avatars.githubusercontent.com/u/23487320?v=4",
        "events_url": "https://api.github.com/users/weiji14/events{/privacy}",
        "followers_url": "https://api.github.com/users/weiji14/followers",
        "following_url": "https://api.github.com/users/weiji14/following{/other_user}",
        "gists_url": "https://api.github.com/users/weiji14/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/weiji14",
        "id": 23487320,
        "login": "weiji14",
        "node_id": "MDQ6VXNlcjIzNDg3MzIw",
        "organizations_url": "https://api.github.com/users/weiji14/orgs",
        "received_events_url": "https://api.github.com/users/weiji14/received_events",
        "repos_url": "https://api.github.com/users/weiji14/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/weiji14/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/weiji14/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/weiji14"
    },
    "comments": 8,
    "comments_content": [
        {
            "author_association": "MEMBER",
            "body": "I'll get on Windows and work on this today (and also #717). The CI failures are getting too annoying.",
            "created_at": "2021-01-31T20:55:47Z",
            "html_url": "https://github.com/GenericMappingTools/pygmt/issues/801#issuecomment-770449352",
            "id": 770449352,
            "issue_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/801",
            "node_id": "MDEyOklzc3VlQ29tbWVudDc3MDQ0OTM1Mg==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/770449352/reactions"
            },
            "updated_at": "2021-01-31T20:55:47Z",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/770449352",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/23487320?v=4",
                "events_url": "https://api.github.com/users/weiji14/events{/privacy}",
                "followers_url": "https://api.github.com/users/weiji14/followers",
                "following_url": "https://api.github.com/users/weiji14/following{/other_user}",
                "gists_url": "https://api.github.com/users/weiji14/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/weiji14",
                "id": 23487320,
                "login": "weiji14",
                "node_id": "MDQ6VXNlcjIzNDg3MzIw",
                "organizations_url": "https://api.github.com/users/weiji14/orgs",
                "received_events_url": "https://api.github.com/users/weiji14/received_events",
                "repos_url": "https://api.github.com/users/weiji14/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/weiji14/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/weiji14/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/weiji14"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "The \"windows-latest - GMT master\" workflow is failing at the `pygmt` install step now, see https://github.com/GenericMappingTools/pygmt/runs/1803109048?check_suite_focus=true#step:10:796. Edit: Was an intermittent issue, should be fixed as `cftime==1.4.0` wheels for Windows are now available at https://pypi.org/project/cftime/1.4.0/#files\r\n\r\n<details>\r\n\r\n```\r\nBuilding wheels for collected packages: pygmt, cftime\r\n  Building wheel for pygmt (PEP 517): started\r\n  Building wheel for pygmt (PEP 517): finished with status 'done'\r\n  Created wheel for pygmt: filename=pygmt-0.2.2.dev90+g8e2b369-py3-none-any.whl size=6820979 sha256=c2d1af0eeaa17c1b6bf2a35d357d35af6377fc8a5cd79d62f5e54021c53257be\r\n  Stored in directory: c:\\users\\runneradmin\\appdata\\local\\pip\\cache\\wheels\\5f\\1d\\d8\\0ec77787b2f8db5f5eb3c9173c0eeaded3f6a35efdaae2a6d5\r\n  Building wheel for cftime (PEP 517): started\r\n  Building wheel for cftime (PEP 517): finished with status 'error'\r\n  ERROR: Command errored out with exit status 1:\r\n   command: 'C:\\Miniconda3\\envs\\test\\python.exe' 'C:\\Miniconda3\\envs\\test\\lib\\site-packages\\pip\\_vendor\\pep517\\_in_process.py' build_wheel 'C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\tmpy8_5atiw'\r\n       cwd: C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-install-cx5f1nhp\\cftime_6eb00282ca204302be377e9d56ab8435\r\n  Complete output (27 lines):\r\n  running bdist_wheel\r\n  running build\r\n  running build_py\r\n  creating build\r\n  creating build\\lib.win-amd64-3.9\r\n  creating build\\lib.win-amd64-3.9\\cftime\r\n  copying src\\cftime\\__init__.py -> build\\lib.win-amd64-3.9\\cftime\r\n  running build_ext\r\n  building 'cftime._cftime' extension\r\n  creating build\\temp.win-amd64-3.9\r\n  creating build\\temp.win-amd64-3.9\\Release\r\n  creating build\\temp.win-amd64-3.9\\Release\\Users\r\n  creating build\\temp.win-amd64-3.9\\Release\\Users\\runneradmin\r\n  creating build\\temp.win-amd64-3.9\\Release\\Users\\runneradmin\\AppData\r\n  creating build\\temp.win-amd64-3.9\\Release\\Users\\runneradmin\\AppData\\Local\r\n  creating build\\temp.win-amd64-3.9\\Release\\Users\\runneradmin\\AppData\\Local\\Temp\r\nSuccessfully built pygmt\r\nFailed to build cftime\r\n  creating build\\temp.win-amd64-3.9\\Release\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-install-cx5f1nhp\r\n  creating build\\temp.win-amd64-3.9\\Release\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-install-cx5f1nhp\\cftime_6eb00282ca204302be377e9d56ab8435\r\n  creating build\\temp.win-amd64-3.9\\Release\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-install-cx5f1nhp\\cftime_6eb00282ca204302be377e9d56ab8435\\src\r\n  creating build\\temp.win-amd64-3.9\\Release\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-install-cx5f1nhp\\cftime_6eb00282ca204302be377e9d56ab8435\\src\\cftime\r\n  C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Tools\\MSVC\\14.28.29333\\bin\\HostX86\\x64\\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -DNPY_NO_DEPRECATED_API=NPY_1_7_API_VERSION -IC:\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-build-env-1pu1oz50\\overlay\\Lib\\site-packages\\numpy\\core\\include -IC:\\Miniconda3\\envs\\test\\include -IC:\\Miniconda3\\envs\\test\\include -IC:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Tools\\MSVC\\14.28.29333\\ATLMFC\\include -IC:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Tools\\MSVC\\14.28.29333\\include -IC:\\Program Files (x86)\\Windows Kits\\NETFXSDK\\4.8\\include\\um -IC:\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.19041.0\\ucrt -IC:\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.19041.0\\shared -IC:\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.19041.0\\um -IC:\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.19041.0\\winrt -IC:\\Program Files (x86)\\Windows Kits\\10\\include\\10.0.19041.0\\cppwinrt /TcC:\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-install-cx5f1nhp\\cftime_6eb00282ca204302be377e9d56ab8435\\src\\cftime\\_cftime.c /Fobuild\\temp.win-amd64-3.9\\Release\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-install-cx5f1nhp\\cftime_6eb00282ca204302be377e9d56ab8435\\src\\cftime\\_cftime.obj\r\n  _cftime.c\r\n  C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-install-cx5f1nhp\\cftime_6eb00282ca204302be377e9d56ab8435\\src\\cftime\\_cftime.c(16852): warning C4996: '_PyUnicode_get_wstr_length': deprecated in 3.3\r\n  C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-install-cx5f1nhp\\cftime_6eb00282ca204302be377e9d56ab8435\\src\\cftime\\_cftime.c(17046): warning C4996: '_PyUnicode_get_wstr_length': deprecated in 3.3\r\n  C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Tools\\MSVC\\14.28.29333\\bin\\HostX86\\x64\\link.exe /nologo /INCREMENTAL:NO /LTCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:C:\\Miniconda3\\envs\\test\\libs /LIBPATH:C:\\Miniconda3\\envs\\test\\PCbuild\\amd64 /LIBPATH:C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Tools\\MSVC\\14.28.29333\\ATLMFC\\lib\\x64 /LIBPATH:C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Tools\\MSVC\\14.28.29333\\lib\\x64 /LIBPATH:C:\\Program Files (x86)\\Windows Kits\\NETFXSDK\\4.8\\lib\\um\\x64 /LIBPATH:C:\\Program Files (x86)\\Windows Kits\\10\\lib\\10.0.19041.0\\ucrt\\x64 /LIBPATH:C:\\Program Files (x86)\\Windows Kits\\10\\lib\\10.0.19041.0\\um\\x64 /EXPORT:PyInit__cftime build\\temp.win-amd64-3.9\\Release\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-install-cx5f1nhp\\cftime_6eb00282ca204302be377e9d56ab8435\\src\\cftime\\_cftime.obj /OUT:build\\lib.win-amd64-3.9\\cftime\\_cftime.cp39-win_amd64.pyd /IMPLIB:build\\temp.win-amd64-3.9\\Release\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-install-cx5f1nhp\\cftime_6eb00282ca204302be377e9d56ab8435\\src\\cftime\\_cftime.cp39-win_amd64.lib\r\n  LINK : fatal error LNK1104: cannot open file 'build\\temp.win-amd64-3.9\\Release\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-install-cx5f1nhp\\cftime_6eb00282ca204302be377e9d56ab8435\\src\\cftime\\_cftime.cp39-win_amd64.exp'\r\n  error: command 'C:\\\\Program Files (x86)\\\\Microsoft Visual Studio\\\\2019\\\\Enterprise\\\\VC\\\\Tools\\\\MSVC\\\\14.28.29333\\\\bin\\\\HostX86\\\\x64\\\\link.exe' failed with exit code 1104\r\n  ----------------------------------------\r\n  ERROR: Failed building wheel for cftime\r\nERROR: Could not build wheels for cftime which use PEP 517 and cannot be installed directly\r\nError: Process completed with exit code 1.\r\n```\r\n\r\n</details>",
            "created_at": "2021-02-01T00:52:13Z",
            "html_url": "https://github.com/GenericMappingTools/pygmt/issues/801#issuecomment-770491937",
            "id": 770491937,
            "issue_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/801",
            "node_id": "MDEyOklzc3VlQ29tbWVudDc3MDQ5MTkzNw==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/770491937/reactions"
            },
            "updated_at": "2021-02-02T08:23:02Z",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/770491937",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/23487320?v=4",
                "events_url": "https://api.github.com/users/weiji14/events{/privacy}",
                "followers_url": "https://api.github.com/users/weiji14/followers",
                "following_url": "https://api.github.com/users/weiji14/following{/other_user}",
                "gists_url": "https://api.github.com/users/weiji14/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/weiji14",
                "id": 23487320,
                "login": "weiji14",
                "node_id": "MDQ6VXNlcjIzNDg3MzIw",
                "organizations_url": "https://api.github.com/users/weiji14/orgs",
                "received_events_url": "https://api.github.com/users/weiji14/received_events",
                "repos_url": "https://api.github.com/users/weiji14/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/weiji14/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/weiji14/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/weiji14"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "If we can't fix it before v0.3.0. I think the workaround would be codes like\r\n```\r\ntry:\r\n    os.remove(self.name)\r\nexcept PermissionError:\r\n    print(\"PyGMT can't delete the temporary file 'xxx' due to unknown causes.\")\r\n```\r\n",
            "created_at": "2021-02-06T03:26:22Z",
            "html_url": "https://github.com/GenericMappingTools/pygmt/issues/801#issuecomment-774389797",
            "id": 774389797,
            "issue_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/801",
            "node_id": "MDEyOklzc3VlQ29tbWVudDc3NDM4OTc5Nw==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/774389797/reactions"
            },
            "updated_at": "2021-02-06T03:26:22Z",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/774389797",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/3974108?v=4",
                "events_url": "https://api.github.com/users/seisman/events{/privacy}",
                "followers_url": "https://api.github.com/users/seisman/followers",
                "following_url": "https://api.github.com/users/seisman/following{/other_user}",
                "gists_url": "https://api.github.com/users/seisman/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/seisman",
                "id": 3974108,
                "login": "seisman",
                "node_id": "MDQ6VXNlcjM5NzQxMDg=",
                "organizations_url": "https://api.github.com/users/seisman/orgs",
                "received_events_url": "https://api.github.com/users/seisman/received_events",
                "repos_url": "https://api.github.com/users/seisman/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/seisman/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/seisman/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/seisman"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "@weiji14 I think the big issue in GMT dev was just fixed in https://github.com/GenericMappingTools/gmt/pull/4777. \r\n\r\nCould you please bump conda-forge's GMT dev version (https://github.com/conda-forge/gmt-feedstock/tree/devel) again, so that we can have full tests running to make sure all tests pass?",
            "created_at": "2021-02-09T19:43:17Z",
            "html_url": "https://github.com/GenericMappingTools/pygmt/issues/801#issuecomment-776194064",
            "id": 776194064,
            "issue_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/801",
            "node_id": "MDEyOklzc3VlQ29tbWVudDc3NjE5NDA2NA==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/776194064/reactions"
            },
            "updated_at": "2021-02-09T19:43:17Z",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/776194064",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/3974108?v=4",
                "events_url": "https://api.github.com/users/seisman/events{/privacy}",
                "followers_url": "https://api.github.com/users/seisman/followers",
                "following_url": "https://api.github.com/users/seisman/following{/other_user}",
                "gists_url": "https://api.github.com/users/seisman/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/seisman",
                "id": 3974108,
                "login": "seisman",
                "node_id": "MDQ6VXNlcjM5NzQxMDg=",
                "organizations_url": "https://api.github.com/users/seisman/orgs",
                "received_events_url": "https://api.github.com/users/seisman/received_events",
                "repos_url": "https://api.github.com/users/seisman/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/seisman/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/seisman/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/seisman"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "Ok. Hard to do it on mobile at the conference but I'll do it as soon as I possibly can, maybe in a day or two.",
            "created_at": "2021-02-10T02:51:50Z",
            "html_url": "https://github.com/GenericMappingTools/pygmt/issues/801#issuecomment-776401307",
            "id": 776401307,
            "issue_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/801",
            "node_id": "MDEyOklzc3VlQ29tbWVudDc3NjQwMTMwNw==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/776401307/reactions"
            },
            "updated_at": "2021-02-10T02:51:50Z",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/776401307",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/23487320?v=4",
                "events_url": "https://api.github.com/users/weiji14/events{/privacy}",
                "followers_url": "https://api.github.com/users/weiji14/followers",
                "following_url": "https://api.github.com/users/weiji14/following{/other_user}",
                "gists_url": "https://api.github.com/users/weiji14/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/weiji14",
                "id": 23487320,
                "login": "weiji14",
                "node_id": "MDQ6VXNlcjIzNDg3MzIw",
                "organizations_url": "https://api.github.com/users/weiji14/orgs",
                "received_events_url": "https://api.github.com/users/weiji14/received_events",
                "repos_url": "https://api.github.com/users/weiji14/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/weiji14/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/weiji14/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/weiji14"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "As it's hard for you, I'll manually submit a PR later.",
            "created_at": "2021-02-10T03:06:11Z",
            "html_url": "https://github.com/GenericMappingTools/pygmt/issues/801#issuecomment-776405963",
            "id": 776405963,
            "issue_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/801",
            "node_id": "MDEyOklzc3VlQ29tbWVudDc3NjQwNTk2Mw==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/776405963/reactions"
            },
            "updated_at": "2021-02-10T03:06:11Z",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/776405963",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/3974108?v=4",
                "events_url": "https://api.github.com/users/seisman/events{/privacy}",
                "followers_url": "https://api.github.com/users/seisman/followers",
                "following_url": "https://api.github.com/users/seisman/following{/other_user}",
                "gists_url": "https://api.github.com/users/seisman/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/seisman",
                "id": 3974108,
                "login": "seisman",
                "node_id": "MDQ6VXNlcjM5NzQxMDg=",
                "organizations_url": "https://api.github.com/users/seisman/orgs",
                "received_events_url": "https://api.github.com/users/seisman/received_events",
                "repos_url": "https://api.github.com/users/seisman/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/seisman/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/seisman/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/seisman"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "Done in https://github.com/conda-forge/gmt-feedstock/pull/134, and recent tests with GMT dev pass (https://github.com/GenericMappingTools/pygmt/actions/runs/553525167).",
            "created_at": "2021-02-10T05:17:16Z",
            "html_url": "https://github.com/GenericMappingTools/pygmt/issues/801#issuecomment-776448672",
            "id": 776448672,
            "issue_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/801",
            "node_id": "MDEyOklzc3VlQ29tbWVudDc3NjQ0ODY3Mg==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 1,
                "laugh": 0,
                "rocket": 0,
                "total_count": 1,
                "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/776448672/reactions"
            },
            "updated_at": "2021-02-10T05:17:16Z",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/776448672",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/3974108?v=4",
                "events_url": "https://api.github.com/users/seisman/events{/privacy}",
                "followers_url": "https://api.github.com/users/seisman/followers",
                "following_url": "https://api.github.com/users/seisman/following{/other_user}",
                "gists_url": "https://api.github.com/users/seisman/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/seisman",
                "id": 3974108,
                "login": "seisman",
                "node_id": "MDQ6VXNlcjM5NzQxMDg=",
                "organizations_url": "https://api.github.com/users/seisman/orgs",
                "received_events_url": "https://api.github.com/users/seisman/received_events",
                "repos_url": "https://api.github.com/users/seisman/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/seisman/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/seisman/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/seisman"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "Ok, I'll close this issue since it seems to have been fixed in https://github.com/GenericMappingTools/gmt/pull/4777",
            "created_at": "2021-02-10T20:47:15Z",
            "html_url": "https://github.com/GenericMappingTools/pygmt/issues/801#issuecomment-777024444",
            "id": 777024444,
            "issue_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/801",
            "node_id": "MDEyOklzc3VlQ29tbWVudDc3NzAyNDQ0NA==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/777024444/reactions"
            },
            "updated_at": "2021-02-10T20:47:15Z",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/777024444",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/23487320?v=4",
                "events_url": "https://api.github.com/users/weiji14/events{/privacy}",
                "followers_url": "https://api.github.com/users/weiji14/followers",
                "following_url": "https://api.github.com/users/weiji14/following{/other_user}",
                "gists_url": "https://api.github.com/users/weiji14/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/weiji14",
                "id": 23487320,
                "login": "weiji14",
                "node_id": "MDQ6VXNlcjIzNDg3MzIw",
                "organizations_url": "https://api.github.com/users/weiji14/orgs",
                "received_events_url": "https://api.github.com/users/weiji14/received_events",
                "repos_url": "https://api.github.com/users/weiji14/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/weiji14/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/weiji14/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/weiji14"
            }
        }
    ],
    "comments_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/801/comments",
    "created_at": "2021-01-22T22:30:06Z",
    "events_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/801/events",
    "html_url": "https://github.com/GenericMappingTools/pygmt/issues/801",
    "id": 792347523,
    "labels": [
        {
            "color": "ee0701",
            "default": true,
            "description": "Something isn't working",
            "id": 563309827,
            "name": "bug",
            "node_id": "MDU6TGFiZWw1NjMzMDk4Mjc=",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/labels/bug"
        },
        {
            "color": "e8ff6b",
            "default": false,
            "description": "Bug or missing feature of upstream core GMT",
            "id": 2059525605,
            "name": "upstream",
            "node_id": "MDU6TGFiZWwyMDU5NTI1NjA1",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/labels/upstream"
        }
    ],
    "labels_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/801/labels{/name}",
    "locked": false,
    "milestone": {
        "closed_at": "2021-02-17T00:26:18Z",
        "closed_issues": 149,
        "created_at": "2020-08-16T22:38:18Z",
        "creator": {
            "avatar_url": "https://avatars.githubusercontent.com/u/23487320?v=4",
            "events_url": "https://api.github.com/users/weiji14/events{/privacy}",
            "followers_url": "https://api.github.com/users/weiji14/followers",
            "following_url": "https://api.github.com/users/weiji14/following{/other_user}",
            "gists_url": "https://api.github.com/users/weiji14/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/weiji14",
            "id": 23487320,
            "login": "weiji14",
            "node_id": "MDQ6VXNlcjIzNDg3MzIw",
            "organizations_url": "https://api.github.com/users/weiji14/orgs",
            "received_events_url": "https://api.github.com/users/weiji14/received_events",
            "repos_url": "https://api.github.com/users/weiji14/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/weiji14/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/weiji14/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/weiji14"
        },
        "description": "Implement subplot wrapper (#412), inset (#788), etc.",
        "due_on": "2021-02-14T08:00:00Z",
        "html_url": "https://github.com/GenericMappingTools/pygmt/milestone/3",
        "id": 5777223,
        "labels_url": "https://api.github.com/repos/GenericMappingTools/pygmt/milestones/3/labels",
        "node_id": "MDk6TWlsZXN0b25lNTc3NzIyMw==",
        "number": 3,
        "open_issues": 0,
        "state": "closed",
        "title": "0.3.0",
        "updated_at": "2021-02-17T00:26:18Z",
        "url": "https://api.github.com/repos/GenericMappingTools/pygmt/milestones/3"
    },
    "node_id": "MDU6SXNzdWU3OTIzNDc1MjM=",
    "number": 801,
    "performed_via_github_app": null,
    "reactions": {
        "+1": 0,
        "-1": 0,
        "confused": 0,
        "eyes": 0,
        "heart": 0,
        "hooray": 0,
        "laugh": 0,
        "rocket": 0,
        "total_count": 0,
        "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/801/reactions"
    },
    "repository_url": "https://api.github.com/repos/GenericMappingTools/pygmt",
    "state": "closed",
    "state_reason": "completed",
    "timeline_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/801/timeline",
    "title": "Potential bugs may cause the recent Windows failures with GMT dev builds",
    "updated_at": "2021-02-10T20:47:16Z",
    "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/801",
    "user": {
        "avatar_url": "https://avatars.githubusercontent.com/u/3974108?v=4",
        "events_url": "https://api.github.com/users/seisman/events{/privacy}",
        "followers_url": "https://api.github.com/users/seisman/followers",
        "following_url": "https://api.github.com/users/seisman/following{/other_user}",
        "gists_url": "https://api.github.com/users/seisman/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/seisman",
        "id": 3974108,
        "login": "seisman",
        "node_id": "MDQ6VXNlcjM5NzQxMDg=",
        "organizations_url": "https://api.github.com/users/seisman/orgs",
        "received_events_url": "https://api.github.com/users/seisman/received_events",
        "repos_url": "https://api.github.com/users/seisman/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/seisman/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/seisman/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/seisman"
    }
}